### DISCUSSION TITLE
How can I get the value of window.innerWidth?

### DISCUSSION THREAD

**Discussion by nisa6delgado** [alpinejs/alpine#2045] [1. Help]
I need a variable defined inside x-data to have true or false value according to the window.innerWidth. But it tells me that window.innerWidth is undefined, I suppose because of the context. How can I get the value of window.innerWidth?

**Comment by SimoTod** [comment]
Weird, it should work: https://codepen.io/SimoTod/pen/powrjEG
Maybe you have a typo somewhere
